src/components/Dropdown/index.tsx

```diff
@@ -139,7 +139,11 @@ const Dropdown = <T extends {}>(props: Props<T>) => {
         </button>
         <ul
           {...getMenuProps()}
-          className={classNames(styles['dropdown-list'], styles[listSizeClass])}
+          className={classNames(
+            styles['dropdown-list'],
+            styles[listSizeClass],
+            isOpen ? '' : styles['dropdown-list--hidden']
```

Comment:
here you can just `!isOpen || styles['dropdown-list--hidden']`

Reply:
Got it but it should be an `&&` instead